test/Analysis/ctu-on-demand-parsing.c

```diff
@@ -0,0 +1,75 @@
+// RUN: rm -rf %t && mkdir %t
+// RUN: mkdir -p %t/ctudir2
+// RUN: echo '[{"directory":"%S/Inputs","command":"clang -c ctu-other.c","file":"ctu-other.c"}]' | sed -e 's/\\/\\\\/g' > %t/ctudir2/compile_commands.json
```

Comment:
Why do we need the `sed`?
Will that work on MacOs and on Windows?

Reply:
I have seen this usage in clang-tidy test files, and have conceded to rely on this method, as generated files contain pathnames, and JSON should contain these paths escaped.

Reply:
I am still worried about windows, but let's see.